drivers/ethernet/eth_w5500.c

```diff
@@ -507,9 +535,9 @@ static int w5500_init(const struct device *dev)
 			LOG_ERR("Unable to configure GPIO pin %u", config->reset.pin);
 			return -EINVAL;
 		}
-		gpio_pin_set_dt(&config->reset, 0);
-		k_usleep(500);
 		gpio_pin_set_dt(&config->reset, 1);
+		k_usleep(500);
+		gpio_pin_set_dt(&config->reset, 0);
```

Comment:
please drop this changes as it's already under open discussion.

Reply:
Removed this change from my PR and waiting for the outcome in separate #46469